vim-engine/src/main/kotlin/com/maddyhome/idea/vim/api/VimSearchHelperBase.kt

```diff
@@ -422,16 +439,25 @@ abstract class VimSearchHelperBase : VimSearchHelper {
       }
     } else if (pos == size) {
       res = size - 1
+    } else if (pos <= 0) {
+      res = 0
     }
     return res
   }
 
-  private fun skipSpace(editor: VimEditor, chars: CharSequence, offset: Int, step: Int, size: Int): Int {
+  private fun skipSpace(
+    editor: VimEditor,
+    chars: CharSequence,
+    offset: Int,
+    step: Int,
+    size: Int,
+    matchEmptyLine: Boolean,
```

Comment:
I'd appreciate a small doc mentioning the details of how `matchEmptyLine` supposed to affect the method.

Reply:
This function no longer exists. It was removed while rewriting the motions.